1.8/development/plugins/creating-modifying-templates.md

```diff
@@ -94,3 +94,46 @@ global $user;
 
 $user['favorite_colour'] = 'Blue';
 ```
+
+## Caching Templates
+Don't forget to cache the template. This is important for pages with big load and prevent the database server from extra querys.
+
+### Caching a template
+An simple way to cache an template only at the place where it's needed, is to use the `THIS_SCRIPT` definition.
```

Comment:
"An simple way" --> "A simple way"

The general rule for a vs. an is that if the word after it starts with a vowel, `an` is used. Otherwise, `a` is typically used.

Also, the comma here isn't necessary.  "is to use..." is still part of the same clause.